setup.py

```diff
@@ -33,7 +37,8 @@ class sdist(_sdist):
     def run(self):
         try:
             from Cython.Build import cythonize
-            cythonize(['thriftrw/*/*.pyx'])
+            # TODO list Cython modules explicitly
```

Comment:
any reason you don't do this in the PR now? Seems like any debt you can avoid in setup.py is worth the investment in the long-run.